storage/client/client.go

```diff
@@ -309,10 +330,35 @@ func (c *StorageClient) Blocks(ctx context.Context, r apiTypes.GetConsensusBlock
 	}
 	for res.rows.Next() {
 		var b Block
-		if err := res.rows.Scan(&b.Height, &b.Hash, &b.Timestamp, &b.NumTransactions); err != nil {
+		var metadata json.RawMessage
+		var proposerRow nodeInfoRow
+		var signerRows []nodeInfoRow
+		if err := res.rows.Scan(
+			&b.Height,
+			&b.Hash,
+			&b.Timestamp,
+			&b.NumTransactions,
+			&metadata,
```

Comment:
Currently unused; let's not pull it from the DB in the first place. (I'm supportive of having the raw meta field in the db in case we need to extract more info from it later)

Reply:
woa is it not used?

Reply:
huh did I intend to add that to the response or something